docs/rules/jsx-sort-props.md

```diff
@@ -98,6 +98,16 @@ With `reservedFirst: ["key"]`, the following will **not** warn:
 <Hello key={'uuid'} name="John" ref="ref" />
 ```
 
+### `ignoreTags`
+
+An array of element (tag) names to ignore when checking the properties order.
+
+With `ignoreTags: ["Can"]`, the following will **not** warn:
+
+```jsx
+<Can i="create" a="post" />
```

Comment:
i think rather than ignoring, it would be better to specify a custom ordering that applies to a specific custom component name. What do you think?

Reply:
Well, it makes sense in the case of the CASL's Can component. 

I can also think about using this in [FontAwesome](https://fontawesome.com/how-to-use/on-the-web/using-with/react) components (I'd like to have the `fixedWitdth` after `icon` because it is better to read then: `<FontAwesomeIcon icon="coffee" fixedWidth />`.

However, I'm reluctant because it will be less generic (uou don't always know all possible props that can be used with an element). Moreover, this will make the configuration much more complicated.

Reply:
This is a very very rare use case; I'm not aware of anything in the entire JS ecosystem besides CASL that has a props API like this. I think it's totally fine to be not generic, since I very much doubt there will ever be a second use case (where the order is sentence-like).

In the case of FontAwesome, just ignoring it wouldn't enforce your preference, it'd just remove enforcement - which is strictly less useful, even if the alternative would require more config.